File: dev-page/src/main/java/org/example/page/task/general/user/signup/Service.java
package org.example.page.task.general.user.signup;

class Service {
    private final Request request;

    Service(Request request) {
        this.request = request;
    }

    void signUp(String id, String password) {
        request.signUp(new EntityGeneralUser(id, password));
    }
}

File: dev-page/src/main/kotlin/org/example/page/task/general/user/signup/Service.kt
package org.example.page.task.general.user.signup

import org.example.core.page.application.AbstractService

internal class Service(private val request: Request): AbstractService() {
    fun signUp(id: String?, password: String?) {
        request.signUp(EntityGeneralUser(id, password))
    }
}
